SourceS/stdarg.h

```diff
@@ -0,0 +1,27 @@
+#if defined(__MACH__)
+#ifndef __STDARG_H
+#define __STDARG_H
+
+#ifndef _VA_LIST_T
+typedef __builtin_va_list va_list;
+#define _VA_LIST_T
+#endif
+#define va_start(ap, param) __builtin_va_start(ap, param)
+#define va_end(ap)          __builtin_va_end(ap)
+#define va_arg(ap, type)    __builtin_va_arg(ap, type)
+
+/* GCC always defines __va_copy, but does not define va_copy unless in c99 mode
+ * or -ansi is not specified, since it was not part of C90.
```

Comment:
Is C90 a typo?
```suggestion
 * or -ansi is not specified, since it was not part of C99.
```

Reply:
It's copied verbatim from the system header only with `#ifdef` changed. Here's the original error that I fixed with this:

```
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.14.sdk/usr/include/sys/_types/
_va_list.h:32:26: error: typedef redefinition with different types ('__darwin_va_list' (aka 'void *') vs '__builtin_va_list')
typedef __darwin_va_list va_list;
                         ^
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/clang/10.0.0/include/
stdarg.h:30:27: note: previous definition is here
typedef __builtin_va_list va_list;
                          ^
1 error generated.
```

Turns out the system `stdarg.h` uses `#ifndef _VA_LIST` instead of `#ifndef _VA_LIST_T`, which introduces the conflicting definitions. I can update that comment text as well if it helps in any way.

Reply:
If it's copied from official headers then just leave it as is